cmd/post.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"bytes"
+	"os"
+	//"encoding/json"
+	"bufio"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// postCmd represents the post command
+var postCmd = &cobra.Command{
+	Use:   "post",
+	Short: "Make POST request to the given URL",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := &http.Client{}
+		//Recieve url from user
+		url := args[0]
+		//Recieve input data from user
+		scanner := bufio.NewScanner(os.Stdin)
+
+		postBody := bytes.NewBuffer(nil)
+		for scanner.Scan() {
```

Comment:
The `json` flag has not yet been implemented.
Also validate the `json` input before proceeding to the next step.

The `putCmd` should be corrected in the similar manner.